File: C/helloWorld.c
#include <stdio.h>
main() {
  long long P = 1,
            E = 2,
            T = 5,
            A = 61,
            L = 251,
            N = 3659,
            R = 271173410,
            G = 1479296389,
            x[] = { G * R * E * E * T , P * L * A * N * E * T };
  puts((char*)x);
}

File: Elixir/HelloWorld.ex
defmodule SayHello do
  @moduledoc """
  A SayHello module that simply prints hello world.
  """

  @doc """
  English implementation of an hello world.

    ## Examples

        iex> SayHello.in_english
        Hello World!
        :ok
  """
  def in_english do
    IO.puts "Hello World!"
  end

  @doc """
  Italian implementation of an hello world.

    ## Examples

        iex> SayHello.in_italian
        Ciao Mondo!
        :ok
  """
  def in_italian do
    IO.puts "Ciao Mondo!"
  end
  
  @doc """
  Spanish implementation of an hello world.

    ## Examples

        iex> SayHello.in_spanish
        Ciao Mondo!
        :ok
  """
  def in_spanish do
    IO.puts "Hola Mundo!"
  end
end
